### ISSUE TITLE
Improve DApp Browser UI with our own landing page instead of stateofdapps

### ISSUE THREAD

**Issue by zhangzhongnan928** [AlphaWallet/alpha-wallet-ios#875]

**Comment by farrahfazirah** [comment]
The storyboarding reference:  https://invis.io/Z5P16OVRM42

The design handover for developers (iOS): https://zpl.io/a3lmrPW

**Comment by JamesSmartCell** [comment]
Looks very clean and fits with app UI, I wonder though if it would be good to have some dapps the user can click on on the page itself?

Maybe not; most search engine front pages before google were full of things to click on, now looking at yahoo it's almost painful.

Still google do have a 'daily feature'. Maybe we can have just one app featured on the page somehow?

Are users ready for the 'google' approach? Maybe we have to go through a stage where we feature a dapp each day that we know works well with the wallet?

**Comment by zhangzhongnan928** [comment]
The DApp Browser home page is like the home page on your mobile.
You can add your own Dapps on that page.
Want to see suggested DApps, click the Discover DApps.




Regards,
[image: AlphaWallet] <https://awallet.io/>

*Victor Zhang*

[image: AlphaWallet Github] <https://github.com/alpha-wallet> [image:
AlphaWallet Twitter] <https://twitter.com/Alpha_wallet> [image: AlphaWallet
Reddit] <https://www.reddit.com/r/AlphaWallet> [image: AlphaWallet Facebook]
<https://www.facebook.com/AlphaWallet/> [image: AlphaWallet Youtube]
<https://www.youtube.com/channel/UCPP12PQeQmQ4_6qXzNvZw-w/> [image:
AlphaWallet Linkedin] <https://www.linkedin.com/company/%CE%B1wallet/> [image:
AlphaWallet Weibo] <https://weibo.com/AlphaWallet>
Join our Telegram group. <https://t.me/AlphaWalletGroup>

**Comment by JamesSmartCell** [comment]
OK that makes sense. Would we want to feature a 'dapp of the day' picked by someone in the team because they liked it?

**Comment by zhangzhongnan928** [comment]
Haha, it's good idea. Android is fine, you can even show the suggested
DApps directly in the home page.
But for iOS no, Apple not allow, we need to hide the suggested Dapps.


Regards,
[image: AlphaWallet] <https://awallet.io/>

*Victor Zhang*

[image: AlphaWallet Github] <https://github.com/alpha-wallet> [image:
AlphaWallet Twitter] <https://twitter.com/Alpha_wallet> [image: AlphaWallet
Reddit] <https://www.reddit.com/r/AlphaWallet> [image: AlphaWallet Facebook]
<https://www.facebook.com/AlphaWallet/> [image: AlphaWallet Youtube]
<https://www.youtube.com/channel/UCPP12PQeQmQ4_6qXzNvZw-w/> [image:
AlphaWallet Linkedin] <https://www.linkedin.com/company/%CE%B1wallet/> [image:
AlphaWallet Weibo] <https://weibo.com/AlphaWallet>
Join our Telegram group. <https://t.me/AlphaWalletGroup>

**Comment by hboon** [comment]
@farrahfazirah I made some minor tweaks for consistency.

I haven't changed this since this is more prominent: in the screen to edit Dapps, shall we use a table instead? (eg. how the build in Contacts or Safari's bookmarks are edited).

@zhangzhongnan928 I remember you said it's nice to share the bookmarks across all wallets. I haven't done that yet (partly because it's just nice that this change can be separate). Once the new Dapp browser is merged into master, can you test this out and let me know which ones should be shared? They are My Dapps and Browsing History. Have created #1000 to track this.

@zhangzhongnan928 we also need a list of dapps, categorised, with these fields:

Name, Description, URL, Category.

e.g name: "AirSwap", description: "Peer-to-Peer trading on Ethereum", url: "https://airswap.io", category: "Games"

Have created #1001 to track this.
